javaparser-core/src/main/java/com/github/javaparser/ast/stmt/CatchClause.java

```diff
@@ -68,6 +69,7 @@ public CatchClause(final Range range,
 		v.visit(this, arg);
 	}
 
+    // TODO Should we deprecate those ?
 	public BlockStmt getCatchBlock() {
```

Comment:
Sure, throw them out, it's violating DRY anyway (we're using the word "catch" twice in catchClause.getCatchBlock() )